### PR TITLE
[ci] Milestone and Label are required before the PR merged

### PR DESCRIPTION
**Pull request by zhongjiajie**
To make our release process easier, I plan to mark milestones and labels required before the PR merged, they use for

* **label**: In this patch, we required a label with at least one 'feature|bug|improvement|document|chore' to make our changelog more convenient
* **milestone**: For easier to cherry-pick PR, currently some of our milestones only exist in issue and not exists in related PR, it is difficult to find those PR bucause github do not have existing API to get them, we can only use timeline API to find them

### CODE REVIEW

**Review by SbloodyS — APPROVED** [review]
That's Great!

**Comment by zhongjiajie** [comment]
I already test in my fork of dolphinscheduler https://github.com/zhongjiajie/dolphinscheduler/pull/8 and it work fine

**Comment by zhongjiajie** [comment]
And I raise a proposal on mailing list https://lists.apache.org/thread/7s0w7h8ozpz72omlnzlc8vwltnkd0m7z this PR should not merge until proposal pass

**Comment by fuchanghai** [comment]
great

> That's Great!

+1

**Comment by zhongjiajie** [comment]
> And I raise a proposal on mailing list https://lists.apache.org/thread/7s0w7h8ozpz72omlnzlc8vwltnkd0m7z this PR should not merge until proposal pass

It seems we do not have -1 in the mail thread, but we have others to discuss like whether we should ask contributors to cherry-pick PR to release the version, So I will merge it into branch dev. Thanks all

**Comment by kezhenxu94** [comment]
Just FYI, existing PRs need to be rebased so the CI checks can pass, committers can help rebase for those approved PRs and merge them if the original authors don't do that in reasonable time.

**Pull request merged**
